src/classes/project_data.py

```diff
@@ -482,7 +482,7 @@ def read_legacy_project_file(self, file_path):
                             failed_files.append(item.name)
 
                 # Delete all tracks
-                track_list = copy.deepcopy(Track.filter())
+                track_list = json.loads(json.dumps(Track.filter()))
```

Comment:
Could filter()  produce a result that cannot be dumped.

Reply:
Yup, this line does not need the json.loads/json.dumps - because the below calls to `.delete()` don't actually modify anything.